### PR TITLE
Update cors-issues.mdx

### PR DESCRIPTION
**Pull request by snowypines**
New CORS Issues

## Summary of change
Added one more example on CORS Issues - based on https://discord.com/channels/603466164219281420/1027876763054899210/1027879470419419157

### CODE REVIEW

File: v2/emailpassword/troubleshooting/cors-issues.mdx [hunk 1]
```diff
@@ -188,6 +188,35 @@ app.use(
 );
 ```
 
+<br />
+
+### `[Error] from origin 'http://localhost:3000' has been blocked by CORS policy ... the value of 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's crednetial mode is 'include'`
+
+This is a pretty long error message but there are two key takeaways here. 
+
+First, similar to our previous issue, this appears when `Access-Control-Allow-Origin` header includes `*`. To fix, add in the correct origin value, http://localhost:3000 in this case, in the CORS settings.
```
Comment (rishabhpoddar) [inline v2/emailpassword/troubleshooting/cors-issues.mdx:197]:
```suggestion
First, similar to our previous issue, this appears when `Access-Control-Allow-Origin` header includes `*`. To fix, add in the correct origin value, `http://localhost:3000` in this case, in the CORS settings.
```

File: v2/emailpassword/troubleshooting/cors-issues.mdx [hunk 2]
```diff
@@ -188,6 +188,35 @@ app.use(
...
+
+Second, for services that come back with a different origin URL each time like Google auth, we'd want to use the wildcard in the following manner - `https://*.domain.com` .
```
Comment (rishabhpoddar) [inline v2/emailpassword/troubleshooting/cors-issues.mdx:199]: I don't understand this, which means probably no one will. Can you be more clear about this? Or just remove this second point if you aren't sure about it

File: v2/emailpassword/troubleshooting/cors-issues.mdx [hunk 3]
```diff
@@ -188,6 +188,35 @@ app.use(
...
+      //highlight-start
+      origin: "http://localhost:3000",  // Make sure this is the same as the website domain
+      //highlight-end
+      allowedHeaders: ["content-type", ...supertokens.getAllCORSHeaders()],
+      methods: ["GET", "PUT", "POST", "DELETE"],
+      credentials: true,
+   })
+);
+```
+
```
Comment (rishabhpoddar) [inline v2/emailpassword/troubleshooting/cors-issues.mdx:219]: Can you also run `npm run start`? That will propogate these changes to other recipe docs as well.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: v2/emailpassword/troubleshooting/cors-issues.mdx (hunk 2 above)
Comment (snowypines) [reply v2/emailpassword/troubleshooting/cors-issues.mdx:199]: Hmm, ok I can remove the second point.

**Comment by rishabhpoddar** [comment]
Please fix conflicts.

**Pull request closed without merging**